crates/ui/src/components/title_bar/linux_window_controls.rs

```diff
@@ -49,6 +53,7 @@ impl RenderOnce for LinuxWindowControls {
                 "minimize",
                 TitlebarButtonType::Minimize,
                 button_hover_color,
+                self.close_window_action.boxed_clone(),
```

Comment:
Is it fine to pass this down to all buttons? Making it optional would clutter up the code more.

Reply:
I think so!